readme.md

```diff
@@ -206,6 +215,57 @@ project. for available options.
 
 **Example usage**: `nativeImageOutput := file("target") / "my-binary"`
 
+### `nativeImageInstalled`
+
+**Type**: `SettingKey[Boolean]`
+
+**Description**: whether GraalVM is manually installed or should be downloaded with coursier.
+
+**Default**: checks if `NATIVE_IMAGE_INSTALLED` / `GRAALVM_INSTALLED` environment variables or
+`native-image-installed` / `graalvm-installed` properties are set to `true`.
+
+### `nativeImageGraalHome`
+
+**Type**: `TaskKey[Path]`
+
+**Description**: path to GraalVM home directory.
+
+**Default**: if `nativeImageInstalled` is `true`, then tries to read the path from
+environment variables 1) `GRAAL_HOME`, 2) `GRAALVM_HOME` or 3) `JAVA_HOME` (in given order).
+Otherwise, automatically installs GraalVM via [Coursier](https://get-coursier.io/).
+Customize this setting if you prefer to not to use environment variables.
+
+**Example usage**: `nativeImageGraalHome := file("/path/to/graalvm/base/directory").toPath`
+
+### `nativeImageRunAgent`
+
+**Type**: `InputKey[Unit]`
+
+**Description**: run application, tracking all usages of dynamic features of an execution with
+[`native-image-agent`][assisted-configuration-of-native-image-builds].
+
+**Example usage**: `sbt myProject/nativeImageRunAgent arg1 arg2`
```

Comment:
I had to look at the scripted test to understand how to use this new feature. I think it would be helpful if the readme included an end-to-end example.
````suggestion
**Example usage**:
```
# Step 0: Start sbt shell.
$ sbt
# Step 1: Run application on the JVM with native-image agent.
> myProject/nativeImageRunAgent arg1 arg2 
# Step 2: Create native-image binary with assisted configuration.
> myProject/nativeImage
# Step 3: Run native-image that was generated with assisted configuration.
> myProject/nativeImageRun arg1 arg2
```
````